docs/snippets/common/csf-3-example-starter.js.mdx

```diff
@@ -0,0 +1,8 @@
+```js
+// CSF 3
+import { Button } from './Button';
+
+export default { component: Button };
```

Comment:
I could not find an example for `web-components`, which as of this comment will use this `common` snippet.

Would appreciate if someone can confirm that is correct.

Reply:
All of the renderer-independent stories use `default.component` for `web-components`.

Example: https://github.com/storybookjs/storybook/blob/next/code/addons/actions/template/stories/basics.stories.ts#L6

And you can see that value getting set up here: https://github.com/storybookjs/storybook/blob/next/code/renderers/web-components/template/components/index.js#L9

Users wouldn't use this much indirection. Here's a more literal example (that happens not to use the default render function, but could): https://github.com/storybookjs/storybook/blob/next/code/renderers/web-components/template/stories/demo-wc-card.stories.js#L5